core/src/main/java/c/org/rajawali3d/scene/SceneModifier.java

```diff
@@ -1,21 +0,0 @@
-package c.org.rajawali3d.scene;
-
-import android.support.annotation.NonNull;
-import c.org.rajawali3d.annotations.RequiresWriteLock;
-
-/**
- * Interface which must be implemented in order to modify a {@link Scene}.
- *
- * @author Jared Woolston (Jared.Woolston@gmail.com)
- */
-public interface SceneModifier {
```

Comment:
Can you help me track what this turned into?

Reply:
Nothing.  Or rather, whatever the client would have wanted to do under this should be done in a RenderTask, along with any changes to its dependent SceneViews or to other Scenes/Sceneviews as needed by their app's logic.  All part of getting away from fine-grained updates and providing atomic transactions of any granularity